tests/time_evol.jl

```diff
@@ -0,0 +1,72 @@
+using ITensors
+
+# We are simulating the time evolution of a 1D spin chain with N sites, where each site is a spin-1/2 particle. 
+# The simulation is done by applying a sequence of unitary gates to an initial state of the system, 
+# which is a product state where each site alternates between up and down.
+let
+  N = 100 # number of sites
+  cutoff = 1E-8 # specifies a truncation threshold for the SVD in MPS representation
+  tau = 0.1 # time step 
+  ttotal = 5.0 # total time of evolution 
+
+  # Make an array of 'site' indices
+  # conserve_qns=true specifies the quantum numbers (total spin in sys = quantum number "S")
+  # that are conserved by the system (as sys evolves)
+  s = siteinds("S=1/2", N; conserve_qns=false)  
+
+  # Make gates (1,2),(2,3),(3,4),... i.e. unitary gates which act on neighboring pairs of sites in the chain
```

Comment:
Comment that this operator does not actually entangle pairs, but rather just acts as a single-particle operator. The language is still in terms of a pair operator, but the operator for one of the pairs is the identity, so it is effectively a one-particle operator.